candle-pyo3/src/lib.rs

```diff
@@ -101,28 +102,6 @@ impl PyDevice {
     }
 }
 
-impl<'source> FromPyObject<'source> for PyDevice {
```

Comment:
The goal here was to be able to specify the device as a string. Could we maintain this functionality?